build-system/global-configs/client-side-experiments-config.json

```diff
@@ -31,6 +31,13 @@
         "00",
         "03"
       ]
+    },
+    {
+      "name": "story-ad-page-outlink",
+      "percentage": 0.01,
+      "rtvPrefix": [
+        "04"
```

Comment:
This will cause this experiment to only run on the Nightly channel
If you want it to be available for Experimental and Beta, add both "00" and "03"

Right now we don't support setting a different percentage for different RTV prefixes, didn't think of this use case!

Reply:
Oh! I missed that there's already an experiment defined with the same. Like I said, that's not supported, and will fail to deploy :/

Reply:
ah we have `00` & `03` above from a previous PR. Ok @jshamble  lets maybe just run at 2% of everything. @danielrozenberg do we need to explicitly need to exclude lts?